lib/cli.js

```diff
@@ -213,6 +214,24 @@ exports.run = async () => { // eslint-disable-line complexity
 
 	const match = arrify(conf.match);
 	const resolveTestsFrom = cli.input.length === 0 ? projectDir : process.cwd();
+
+	const files = cli.input.map(file => path.relative(resolveTestsFrom, path.resolve(process.cwd(), file)));
+
+	for (const file of cli.input) {
```

Comment:
```suggestion
      for (const file of files) {
```

@novemberborn Right? If not, I suggest the mapping from `cli.input` to `files` move below this block (as before).

Reply:
@ulken Yes good catch. Would you like to open a PR?